wagtail/contrib/modeladmin/templates/modeladmin/index.html

```diff
@@ -16,15 +16,20 @@
 {% block content %}
     {% block header %}
         <header class="nice-padding hasform">
-            <div class="row header-title">
-                <div class="left">
-                    <div class="col">
-                        {% block h1 %}<h1 {% if view.header_icon %}class="icon icon-{{ view.header_icon }}"{% endif %}>{{ view.get_page_title }}<span></span></h1>{% endblock %}
+            <div class="row">
+                <div class="left header-left">
+                    <div class="col header-title">
+                        {% block h1 %}
+                        <h1 {% if view.header_icon %}class="icon icon-{{ view.header_icon }}"{% endif %}>
+                            {{ view.get_page_title }}
+                            {% include 'modeladmin/includes/result_count.html' %}<span></span>
```

Comment:
Is the trailing blank span here intentional.

Just realised it's also in the original. Perhaps it's intended to be directly next to the title content instead of here? Or it's a typo.

Reply:
Yeah. I have no idea. I didn't want to remove it, since I don't know why it is there, but it is annoying as heck.